bugsnag-kmp/src/jsMain/kotlin/com/bugsnag/kmp/JsEvent.kt

```diff
@@ -0,0 +1,21 @@
+@file:JsModule("@bugsnag/core")
+
+package com.bugsnag.kmp
+
+@JsName("Event")
+public external class JsEvent {
+    public constructor(errorClass: String, errorMessage: String)
+    public var apiKey: String
```

Comment:
`event.apiKey` is optional in JS

```suggestion
    public var apiKey: String?
```